dmscripts/compare_dmp_and_dnb_data.py

```diff
@@ -0,0 +1,324 @@
+import datetime
+import numpy as np
+import requests
+import unicodecsv
+
+from dmapiclient import DataAPIClient
+from dmscripts.helpers.auth_helpers import get_auth_token
+from dmutils.env_helpers import get_api_endpoint_from_stage
+
+
+def levenshtein_ratio_and_distance(s, t, ratio_calc=False):
```

Comment:
are we really going to have our own implementation of this instead of adding e.g. `python-levenshtein` or even `fuzzywuzzy` to the deps?

Reply:
> are we really going to have our own implementation of this instead of adding e.g. `python-levenshtein` or even `fuzzywuzzy` to the deps?

I had problems importing `fuzzywuzzy` and the only thing I needed from it was the metric, so I just added it as a function to be able to work on everything else.

Reply:
But everything "just works" with pipi doesn't it?